cypress/support/commands.js

```diff
@@ -25,17 +25,44 @@
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
 import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
+import ArticlePageObject from '../support/pages/articlePageObject';
+
+const articlePage = new ArticlePageObject();
 
 addMatchImageSnapshotCommand();
 
 Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+Cypress.Commands.add('register', (
+  email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
```

Comment:
The `register` command uses default values for email, username, and password. Ensure these defaults are suitable for all test scenarios or consider removing them to force explicit values in each test case.